openapi-cli/src/test/resources/ballerina-to-openapi/expected_gen/response/status_code_errors_02.yaml

```diff
@@ -0,0 +1,162 @@
+openapi: 3.0.1
+info:
+  title: PayloadV
+  version: 0.0.0
+servers:
+  - url: "{server}:{port}/payloadV"
+    variables:
+      server:
+        default: http://localhost
+      port:
+        default: "9000"
+paths:
+  /users/{id}:
+    get:
+      operationId: getUsersId
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/User'
+        "202":
+          description: Accepted
+        "404":
+          description: NotFound
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorInfo'
+        "400":
+          description: BadRequest
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorPayload'
+  /users:
+    post:
+      operationId: postUsers
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UserWithoutId'
+        required: true
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/User'
+        "400":
+          description: BadRequest
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: '#/components/schemas/ErrorInfo'
+                  - $ref: '#/components/schemas/ErrorPayload'
+        "202":
+          description: Accepted
+        "409":
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorInfo'
+  /test1:
+    get:
+      operationId: getTest1
+      responses:
+        default:
+          description: Any Response
+          content:
+            '*/*':
+              schema:
+                description: Any type of entity body
+        "202":
+          description: Accepted
+  /test2:
+    get:
+      operationId: getTest2
+      responses:
+        default:
+          description: Any Response
+          content:
+            '*/*':
```

Comment:
Do not we need to represent header details here since this is `DefaultError` as in here https://github.com/ballerina-platform/ballerina-library/issues/5135#:~:text=Default%20error%20response-,headers%3A,-req%5C%2Did%3A

Reply:
For this test case the status code error details does not contain any header details. So there is not header section in the generated OAS.